docs/index.md

```diff
@@ -3,26 +3,26 @@ layout: page
 title: Documentation
 ---
 
-[Fluo] and [Fluo Recipes] have seperate documentation as they are different repositories with their own release cycle.
+[Apache Fluo] and [Apache Fluo Recipes] have seperate documentation as they are different repositories with their own release cycle.
 
-## Fluo
+## Apache Fluo
 
 Latest release is `{{ site.latest_fluo_release }}` made on {{ site.latest_fluo_release_date }}.
```

Comment:
Since the latest release is currently not an Apache release, need to point this out.